File: DoConnect/src/main/java/cogent/infotech/com/DoConnect/entity/Answer.java
package cogent.infotech.com.DoConnect.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity(name = "Answers")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "Answers")
public class Answer {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String description_answer;
	private String img_src;
	private String status;
	private String datetime;
	
	@ManyToOne
	@JoinColumn(name = "question_id")
	private Question question;
	
	@OneToOne
	private Admin approved_by;
	
	@OneToOne
	private User created_by;
}

File: DoConnect-Front/src/app/model/answer.ts
import {User} from './user'

export class Answer {
    constructor(
        public id: number,
        public description_answer: string,
        public img_src: string,
        public status: string,
        public datetime: string | null,
        public question_id:number,
        public aapproved_by: User,
        public acreated_by: User
    ){}
}
